Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.17.6-bullseye
+
+ENV UID=1000
+ENV GID=1000
+ENV GOXYGEN_DOCKER=true
```

Comment:
I don't quite understand why this part is needed (as well as the corresponding changes in the Go code).

I have tested and the following command would correctly do what we need (given there is a symlink to goxygen executable and the workdir of an image is set to `/app`):
```
docker run --rm -it -v $(pwd):/app image-tag goxygen init my-app
```

Let's optimize it.